ext/odbc/php_odbc.c

```diff
@@ -1736,24 +1736,31 @@ PHP_FUNCTION(odbc_fetch_row)
 PHP_FUNCTION(odbc_result)
 {
 	char *field;
-	zend_string *field_str;
+	zend_string *field_str, *pv_field_str;
+	zend_long pv_field_long;
```

Comment:
@carusogabriel I don't think they are noticably cheaper. The real benefit is being able to accept arguments whose type is validated according to the standard semantics, rather than in a custom way.